doc/script_commands.txt

```diff
@@ -4414,6 +4414,9 @@ The following flag values are accepted:
 Unlike 'setfalcon' and 'setcart' this will not work at all if they aren't of a
 class which can ride a mount.
 
+For clients PACKETVER_MAIN_NUM >= 20191120 || PACKETVER_RE_NUM >= 20191106
+you can pass a type flag which specifies the madogear wanted.
```

Comment:
```suggestion
you can pass a <type> which specifies the madogear wanted.
```

Reply:
perhaps list the madogear type too? 

sometime I find that its troublesome / less user friendly because had to browse `script.c` or the `doc/constants` just to find the constants which user may or may not know what are the constants name to begin with.